src/common/proto-irc.c

```diff
@@ -1026,6 +1027,9 @@ process_named_msg (session *sess, char *type, char *word[], char *word_eol[],
 		ex[0] = 0;
 		safe_strcpy (nick, word[1], sizeof (nick));
 		ex[0] = '!';
+
```

Comment:
Being in this else block seems unclear. This block is about copying nick values.

I assume the reason is because the other block is server messages, but maybe this can be pulled out or at least commented why its here.

Reply:
I moved it out. Now that I think about it there should be a `serv->have_account_tag` check around it anyway.